cas-management-webapp-support/src/main/java/org/jasig/cas/services/web/factory/DefaultRegisteredServiceFactory.java

```diff
@@ -0,0 +1,176 @@
+package org.jasig.cas.services.web.factory;
+
+import org.jasig.cas.services.AbstractRegisteredService;
+import org.jasig.cas.services.RegisteredService;
+import org.jasig.cas.services.RegisteredServiceAccessStrategy;
+import org.jasig.cas.services.RegisteredServiceAttributeReleasePolicy;
+import org.jasig.cas.services.RegisteredServiceProxyPolicy;
+import org.jasig.cas.services.RegisteredServiceUsernameAttributeProvider;
+import org.jasig.cas.services.web.beans.RegisteredServiceEditBean.FormData;
+import org.jasig.cas.services.web.beans.RegisteredServiceEditBean.ServiceData;
+import org.jasig.cas.services.web.beans.RegisteredServiceViewBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+/**
+ * Default implmentation of {@link RegisteredServiceFactory}.
+ *
+ * @author Daniel Frett
+ * @since 4.2
+ */
+@Component
+public final class DefaultRegisteredServiceFactory implements RegisteredServiceFactory {
```

Comment:
Always name components. Implicit behavior could be dangerous. Do this for all please.